config/FindVIGRANUMPY_DEPENDENCIES.cmake

```diff
@@ -93,7 +93,7 @@ IF(PYTHONINTERP_FOUND)
     IF(Boost_FOUND)
         FIND_LIBRARY(Boost_PYTHON_LIBRARY
                      NAMES boost_python-py${PYTHON_VERSION_MAJOR}${PYTHON_VERSION_MINOR}   # Linux with multiple Python versions
-                           boost_python-mt                                                 # Mac
+                           boost_python${PYTHON_VERSION_MAJOR}                             # Mac with Python 3
```

Comment:
I've updated this to include multithreaded libraries as a last resort. In all cases, I have tried to get the correction Python version, if possible.